### PR TITLE
Initial Setup

### PR DESCRIPTION
**Pull request by samhstn**
Addresses #1 

Initial setup:
- [x] Travis
- [x] Code coverage
- [x] Linter
- [x] Beautify Readme (Badges :1st_place_medal:, etc)
- [x] Build first iteration of amp converter
- [x] Publish on [hex](https://hex.pm/)

### CODE REVIEW

**Comment by samhstn** [comment]
Initial documentation published at:
https://hexdocs.pm/amp/0.1.0
Available on hexdocs:
https://hex.pm/packages/amp

File: README.md [hunk 1]
```diff
@@ -1 +1,64 @@
...
+
+[![Build Status](https://travis-ci.org/dwyl/amp-elixir.svg?branch=master)](https://travis-ci.org/dwyl/amp-elixir)
+[![codecov.io](https://codecov.io/github/dwyl/amp-elixir/coverage.svg?branch=master)](https://codecov.io/github/dwyl/amp?branch=master)
+[![Hex version](https://img.shields.io/hexpm/v/amp.svg "Hex version")](https://hex.pm/packages/amp)
+
+Amp transforms your Markdown into AMP-compliant html so it loads super-fast!
+
+## Why?
+
+Google is putting their clout behind the AMP (Accelerated Mobile Pages) Project to help speed up page load times on the Mobile Web: https://googleblog.blogspot.co.uk/2015/10/introducing-accelerated-mobile-pages.html
```
Comment (nelsonic) [inline README.md:11]: http://rhodesmill.org/brandon/2012/one-sentence-per-line

File: README.md [hunk 2]
```diff
@@ -1 +1,64 @@
...
+
+This module lets you convert Markdown (`.md`) files to AMP-compliant `.html` so that you can serve them up to mobile clients much faster.
+
+## What?
+
+For many, reading on the mobile web is a slow, clunky and frustrating experience - but it doesn’t have to be that way. The Accelerated Mobile Pages (AMP) Project is an open source initiative that embodies the vision that publishers can create mobile optimized content once and have it load instantly everywhere.
```
Comment (nelsonic) [inline README.md:17]: long lines are harder to update/review.
keep it to one sentence per line for readability.
http://rhodesmill.org/brandon/2012/one-sentence-per-line

**Review by nelsonic — APPROVED** [review]
@shouston3 this looks _really_ good to me. 
passing to @jruts for second review. 👍

File: README.md (hunk 2 above)
Comment (samhstn) [reply README.md:17]: Thanks for the review, I'll know for next time 👍

**Review by jruts — APPROVED** [review]
Nothing to comment. It looks very good to me :+1:

**Comment by jruts** [comment]
@nelsonic you will have to merge, no rights to do so :)

**Comment by iteles** [comment]
@jruts Thanks for the review! I'll merge this one but just so you know: core team now has admin access to this repo :tada:

**Pull request merged**